### ISSUE TITLE
Prepare release v.3.0.0

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-optmeowt#273]
Once we have finalized the following issues
- mv3: #175, #259, #262, #266
- UI: #267, #268,, #272
- Opt out: #73, #269
we should prepare a new release.

First, each of us will do some informal testing, then @OliverWang13 or @sophieeng, will package the zip and release on GitHub, and I will submit the new version on the Chrome Web Store and  the Mozilla Add-on Store.

**Comment by SebastianZimmeck** [comment]
Is Firefox not following a completely different architecture, i.e., there is no manifest?

**Comment by kalicki1** [comment]
As far as I remember, Firefox does use a similar extension structure to Chrome and uses manifest files, though they do differ on support of a few smaller APIs.

**Comment by SebastianZimmeck** [comment]
There two things that need to happen:
1. What's the situation of MV3 on Firefox? What are we going to do?
2. After the MV3 merge, we need to test whether the functionality is still there, for example, is the domain list properly populated, are the DAA cookies still working, ... ?

@OliverWang13 is on the lead here and @kalicki1 and @sophieeng will help out.

**Comment by SebastianZimmeck** [comment]
@kalicki1 will also work on #262 as part of the general MV3 revision.

**Comment by OliverWang13** [comment]
Yes, @kalicki1 is correct in saying that firefox is not supporting mv3 yet. We could submit an earlier but functioning version of the code (like before merging #175 with main) and that should work. Additionally, I believe we could make a different manifest file that runs our MV3 code as if it is MV2 code, but since we already have a working MV2 version I am not sure if this is needed.

**Comment by SebastianZimmeck** [comment]
OptMeowt's version needs to be updated in the manifest.

**Comment by SebastianZimmeck** [comment]
Once we finalized our mv3 transition, @OliverWang13 will create a GitHub release and deploy on the Firefox store, and I will deploy on the Chrome Web Store.

**Comment by OliverWang13** [comment]
We have made the GitHub release.
